scex-core/src/main/scala/com/avsystem/scex/ExpressionProfile.scala

```diff
@@ -9,7 +9,8 @@ class ExpressionProfile(
   val symbolValidator: SymbolValidator,
   val symbolAttributes: SymbolAttributes,
   val expressionHeader: String,
-  val expressionUtils: NamedSource) {
+  val expressionUtils: NamedSource,
+  val dynamicVariablesEnabled: Boolean) {
```

Comment:
Please add overloaded constructor that preserves the previous signature for binary compatibility (you can annotate with `@bincompat` and make it package private).